From b23194cd0f635d394b560aa9b802744b00f02014 Mon Sep 17 00:00:00 2001
From: Lucio Correia <luciojhc@linux.vnet.ibm.com>
Date: Tue, 14 Feb 2017 17:54:05 -0200
Subject: [PATCH] Change kill mode for wokd service

Python process is not being killed on wokd service restart.
This fixes it by killing whole process tree.

Signed-off-by: Lucio Correia <luciojhc@linux.vnet.ibm.com>
---
 contrib/wokd.service.systemd | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/wokd.service.systemd b/contrib/wokd.service.systemd
index 040f94e..11faa8b 100644
--- a/contrib/wokd.service.systemd
+++ b/contrib/wokd.service.systemd
@@ -8,7 +8,7 @@ After=nginx.service
 Type=simple
 ExecStart=/usr/bin/wokd
 EnvironmentFile=/etc/wok/wok.conf
-KillMode=process
+KillMode=control-group
 
 [Install]
 Alias=wok.service
-- 
2.7.4

